src/openvpn/manage.c

```diff
@@ -137,6 +137,8 @@ man_help(void)
     msg(M_CLIENT, "push-update-broad options : Broadcast a message to update the specified options.");
     msg(M_CLIENT, "                            Ex. push-update-broad \"route something, -dns\"");
     msg(M_CLIENT, "push-update-cid CID options : Send an update message to the client identified by CID.");
+    msg(M_CLIENT, "reload-push-options [sync] : Reload push options from config file for new clients.");
+    msg(M_CLIENT, "                            With 'sync': also sync options to connected clients (add new, remove old).");
```

Comment:
As we are adding new commands, we may need to increment  `MANAGEMENT_VERSION` in `manage.h` so that the client can determine whether the option is supported.

Reply:
MANAGEMENT_VERSION incremented to 6.

one question: is the wording "sync" ok should i rename it to "broadcast"?